src/tests/ngraph_helpers/snippets_ngraph_functions/include/subgraph_roll_matmul_roll.hpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <vector>
+#include "openvino/core/model.hpp"
+#include "snippets_helpers.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+class SubgraphRollMatMulRollFunction : public SnippetsFunctionBase {
+public:
+    SubgraphRollMatMulRollFunction(
+        const std::vector<ov::PartialShape>& input_shapes,
+        const element::Type input_type,
+        const bool transpose,
+        const bool mat_mul);
+
+protected:
+    std::shared_ptr<Model> initOriginal() const override;
+
+private:
+    const bool transpose;
+    const bool mat_mul;
```

Comment:
These two values are never used. Do we really need them?

Reply:
Skipped members are proof that `SnippetsFunctionBase` inheritance only complicates test maintenance during development. Were used before, removed now.